### PR TITLE
Fix code style issues and add more TSLint rules

### PR DESCRIPTION
**Pull request by Acid147**
I fixed most code style issues (mostly curly braces) and added new TSLint rules. Feel free to remove some of the rules I added if you don't like them. I tried to adjust the rules to the existing code style.

-----
[View rendered CHANGELOG.md](https://github.com/Acid147/csharp-colors/blob/master/CHANGELOG.md)
[View rendered README.md](https://github.com/Acid147/csharp-colors/blob/master/README.md)

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
         "sourceMap": true,
         "rootDir": "src",
         /* Strict Type-Checking Option */
-        "strict": false,   /* enable all strict type-checking options */
+        "strict": false,
+        /* enable all strict type-checking options */
```
Comment (Logerfo) [inline tsconfig.json]: This doesn't look right...

File: .vscode/extensions.json [hunk 2]
```diff
@@ -6,4 +6,4 @@
 		"ban.spellright",
 		"eamodio.tsl-problem-matcher"
 	]
-}
+}
```
Comment (Logerfo) [inline .vscode/extensions.json]: Is there a rule for enforcing new lines at the end of files? Is [that](https://palantir.github.io/tslint/rules/eofline/) it? I don't feel comfortable without them.

**Comment by Logerfo** [comment]
I don't know what's happening to FOSSA... I'll keep trying. If it still doesn't work in a while, I'll merge it without the check. Please check my review, though.

**Comment by Acid147** [comment]
No, I haven't added a rule about that. My guess is that I have an extension that reformats JSON, I'll look into it and change it back.

**Comment by Acid147** [comment]
The JSON files should now be the same as before.

I had one extension installed that reformatted JSON code. The removal of final newlines seems to come from the JSON formatter included in Visual Studio Code and there is no setting to turn this feature off other than activating another setting, "Insert Final Newline".

**Comment by Logerfo** [comment]
Thanks for the changes, LGMT. I hope you don't mind if I wait until tomorrow to merge it. It seems like FOSSA API is having some issues, making the travis build fail. If it still doesn't work, I'll merge it anyway.

**Comment by Logerfo** [comment]
FOSSA is still not playing along... moving on despite it.

**Pull request merged**